tests/sentry/api/endpoints/test_group_hashes.py

```diff
@@ -1,28 +1,101 @@
 from __future__ import absolute_import
 
+import copy
+from datetime import timedelta
+from django.utils import timezone
+
 from six.moves.urllib.parse import urlencode
 
 from sentry.models import GroupHash
-from sentry.testutils import APITestCase
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils.factories import DEFAULT_EVENT_DATA
+from sentry.eventstream.snuba import SnubaEventStream
 
 
-class GroupHashesTest(APITestCase):
-    def test_simple(self):
+class GroupHashesTest(APITestCase, SnubaTestCase):
+    def test_only_return_latest_event(self):
         self.login_as(user=self.user)
 
-        group = self.create_group()
-        GroupHash.objects.create(group=group, hash='a' * 32)
-        GroupHash.objects.create(group=group, hash='b' * 32)
+        min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        two_min_ago = (timezone.now() - timedelta(minutes=2)).isoformat()[:19]
+        new_event_id = 'b' * 32
+
+        old_event = self.store_event(
+            data={
+                'event_id': 'a' * 32,
+                'message': 'message',
+                'timestamp': two_min_ago,
+                'stacktrace': copy.deepcopy(DEFAULT_EVENT_DATA['stacktrace']),
+                'fingerprint': ['group-1']
+            },
+            project_id=self.project.id,
+        )
+
+        new_event = self.store_event(
+            data={
+                'event_id': new_event_id,
+                'message': 'message',
+                'timestamp': min_ago,
+                'stacktrace': copy.deepcopy(DEFAULT_EVENT_DATA['stacktrace']),
+                'fingerprint': ['group-1']
+            },
+            project_id=self.project.id,
+        )
+
+        assert new_event.group_id == old_event.group_id
+
+        url = u'/api/0/issues/{}/hashes/'.format(new_event.group_id)
+        response = self.client.get(url, format='json')
+
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 1
+        assert response.data[0]['latestEvent']['eventID'] == new_event_id
 
-        url = u'/api/0/issues/{}/hashes/'.format(group.id)
+    def test_return_multiple_hashes(self):
+        self.login_as(user=self.user)
+
+        min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        two_min_ago = (timezone.now() - timedelta(minutes=2)).isoformat()[:19]
```

Comment:
What's the significance of this magic number (`:19`)? Probably should at least be commented.